.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+    "trailingComma": "all",
+    "tabWidth": 2,
+    "semi": false,
```

Comment:
Pablo and I suggest to keep the semicolons in order to understand the code we write. We are probbably getting lost without them

Reply:
Okei i got it !!